clang_9/Dockerfile

```diff
@@ -13,38 +13,38 @@ ENV LLVM_VERSION=9.0 \
 RUN dpkg --add-architecture i386 \
     && apt-get -qq update \
     && apt-get install -y --no-install-recommends \
-       sudo=1.* \
-       wget=1.* \
-       git=1:2.* \
-       g++-multilib=4:9.* \
+       sudo \
+       wget \
+       git \
+       g++-multilib \
        clang-9=1:9* \
-       make=4.* \
-       libc6-dev-i386=2.* \
-       libgmp-dev=2:6.* \
-       libmpfr-dev=4.* \
-       libmpc-dev=1.* \
-       nasm=2.* \
-       dh-autoreconf=19 \
-       libffi-dev=3.* \
-       libssl-dev=1* \
-       ninja-build=1.* \
-       libc++-dev=1:9.* \
-       libc++abi-dev=1:9.* \
-       pkg-config=0.* \
-       subversion=1.* \
-       zlib1g-dev=1:1.* \
-       libbz2-dev=1.* \
-       libsqlite3-dev=3.* \
-       libreadline-dev=8.* \
-       xz-utils=5.* \
-       curl=7.* \
-       libncurses5-dev=6.* \
-       libncursesw5-dev=6.* \
-       liblzma-dev=5.* \
-       gnupg=2.* \
+       make \
+       libc6-dev-i386 \
+       libgmp-dev \
+       libmpfr-dev \
+       libmpc-dev \
+       nasm \
+       dh-autoreconf \
+       libffi-dev \
+       libssl-dev \
+       ninja-build \
+       libc++-dev \
+       libc++abi-dev \
+       pkg-config \
+       subversion \
+       zlib1g-dev \
+       libbz2-dev \
+       libsqlite3-dev \
+       libreadline-dev \
+       xz-utils \
+       curl \
+       libncurses5-dev \
+       libncursesw5-dev \
+       liblzma-dev \
+       gnupg \
        ca-certificates \
-       # libc++abi-dev:i386=1:9.* \
-       # libc++-dev:i386=1:9.* \
+       libc++abi-dev:i386 \
```

Comment:
Why was this commented out  and why did you enable it again?

Reply:
I see, thanks for pointing. libc++ 9.x is not available, there is a version available, I need to check it.